src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -128,28 +130,56 @@ function filterRows(mode: DisplayMode, rows: AnonymizedResultRow[]) {
   }
 }
 
+function addValue(rowData: TableRowData, values: AnonymizedValue[]) {
```

Comment:
Function name is `addValue` but it adds a list of values. Maybe `addValues` or `addValuesToRow`?